docs/demomaterials/README.md

```diff
@@ -0,0 +1,144 @@
+_Table of contents_
+
+<!-- TOC depthFrom:2 insertAnchor:false orderedList:true updateOnSave:true withLinks:true -->
+
+- [Purpose of this repo](#purpose-of-this-repo)
+- [Demo](#demo)
+  - [Demo Script](#demo-script)
+    - [Basic](#basic)
+    - [Advanced](#advanced)
+  - [Online resources](#online-resources)
+    - [Technology hands-on](#technology-hands-on)
+    - [Videos](#videos)
+  - [Offline resources](#offline-resources)
+    - [Technology hands-on](#technology-hands-on-1)
+- [Additional resources on the technology](#additional-resources-on-the-technology)
+  - [Networking](#networking)
+  - [VM's](#vms)
+    - [Windows workloads](#windows-workloads)
+  - [Console access or UI](#console-access-or-ui)
+  - [Development](#development)
+
+<!-- /TOC -->
+
+## Purpose of this repo
+
+This repository and its contents, aim to provide instructions and materials to be able to host a demo on KubeVirt technology.
+
+It contains links to different materials already available on the website or videos in the KubeVirt channel that could help set up a demonstration environment
```

Comment:
Should the sentence end with punctuation?

Reply:
done in next push